training_config/coref.jsonnet

```diff
@@ -78,11 +78,13 @@
     "spans_per_word": 0.4,
     "max_antecedents": 100
   },
-  "iterator": {
-    "type": "bucket",
-    "sorting_keys": [["text", "num_tokens"]],
-    "padding_noise": 0.0,
-    "batch_size": 1
+  "data_loader": {
+    "batch_sampler": {
+      "type": "bucket",
+      "sorting_keys": [["text", "tokens___token_ids"]],
```

Comment:
This looks different than the original, that is called num_tokens. Is that right?

Reply:
Great, thanks!

Reply:
Except `num_tokens` doesn't work on master, so I think what you had here at first is correct.

Reply:
Though the right thing to do is probably just remove this key altogether.